packages/eslint-plugin/src/rules/unit-naming-rule.test.ts

```diff
@@ -40,15 +40,29 @@ tester.run('unit-naming-rule', unitNamingRule, {
       output: `${ImportAtom}; const someAtom = atom(0, 'someAtom')`,
     },
     {
-      code: `${ImportAtom}; const some = atom(0, 'SomeDomain._some')`,
-      options: [{ atomPostfix: 'Atom' }],
-      errors: [{ messageId: 'postfixMissing' }],
-      output: `${ImportAtom}; const someAtom = atom(0, 'SomeDomain._someAtom')`,
+      code: `${ImportAtom}; function reatomSome() { const field = atom(0, 'reatomSome._unrelated'); }`,
+      errors: [{ messageId: 'nameIncorrect' }],
+      output: `${ImportAtom}; function reatomSome() { const field = atom(0, 'reatomSome._field'); }`,
+    },
+    {
+      code: `${ImportAtom}; function reatomSome() { const field = atom(0, 'field') }`,
+      errors: [{ messageId: 'nameIncorrect' }],
+      output: `${ImportAtom}; function reatomSome() { const field = atom(0, 'reatomSome.field') }`,
+    },
+    {
+      code: `${ImportAtom}; function reatomSome() { const field = atom(0, 'Some.field') }`,
+      errors: [{ messageId: 'nameIncorrect' }],
+      output: `${ImportAtom}; function reatomSome() { const field = atom(0, 'reatomSome.field') }`,
+    },
+    {
+      code: `${ImportAtom}; function reatomSome({name}) { const field = atom(0, 'field'); }`,
+      errors: [{ messageId: 'nameIncorrect' }],
+      output: `${ImportAtom}; function reatomSome({name}) { const field = atom(0, \`\${name}.field\`); }`,
```

Comment:
Please, add additional tests cases (and implementation, if needed) if the fabric is an action (`const reatomSome = action(...)`).